src/hooks/useFlags/flagsmith/useAllFlagsmithFlags.tsx

```diff
@@ -0,0 +1,15 @@
+import { useFlagsmith } from "flagsmith/react";
+import camelCase from "lodash/camelCase";
+
+import { FlagWithName } from "../types";
+
+export const useAllFlagsmithFlags = (): FlagWithName[] => {
+  const flagsmith = useFlagsmith();
+  const flags = flagsmith.getAllFlags();
+
+  return Object.entries(flags).map(([name, { value, enabled }]) => ({
+    name: camelCase(name),
+    value: value || "",
```

Comment:
```suggestion
    value: value ?? "",
```

Reply:
**suggestion (non-blocking)**: Based on the types I can see that `0` or `false` could be legit values here that are going to get overridden. If I'm not mistaken there are couple more lines that need this change.